### ISSUE TITLE
Size limit not working

### ISSUE THREAD

**Issue by GoogleCodeExporter** [napsterrahul1/buddypress-media#18]
```
[What steps will reproduce the problem? (We need detailed, specific steps)]
1. Go to upload new media
2. Select media
3. Upload it

[What is the expected output? What do you see instead?]
I've set limit for file size on 100kb, but managed to upload file which has 
200kb. 

What's wrong with size limiter?

[What version of BP Media are you using?]


[What version of WordPress are you using?]


[What is the URL of your web host?]


[If you are running a public site, what is the URL to a page that shows the
problem?]


[Do you know how to use phpMyAdmin?]


[Do you know how to edit PHP code?]


[Please provide any additional information below]

```

Original issue reported on code.google.com by `tripatw...@gmail.com` on 23 Jul 2010 at 1:13

**Comment by GoogleCodeExporter** [comment]
```
We need to know what version of BP Media, Buddypress, and Wordpress you're 
using before we can try to reproduce this problem.
```

Original comment by `CarlRoett@gmail.com` on 23 Jul 2010 at 9:03

**Comment by GoogleCodeExporter** [comment]
```
1. BP Album+ 0.1.9 nightly build 2010-06-26
2. BuddyPress ver. 1.2.5
3. WP 3.0
```

Original comment by `tripatw...@gmail.com` on 26 Jul 2010 at 10:17

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Fixed with the release of 0.1.8.11
```

Original comment by `fanquake` on 24 Sep 2011 at 11:41
